model/DeliveryLog/SyncDeliveryLogService.php

```diff
@@ -306,4 +309,26 @@ protected function markLogsAsSynced($syncSuccess)
         }
     }
 
+    /**
+     * @param array $events
+     * @param string $boxId
+     * @return bool|void
+     * @throws \common_exception_Error
+     */
+    public function saveBoxId($events, $boxId)
+    {
+        $user = \common_session_SessionManager::getSession()->getUser()->getIdentifier();
```

Comment:
There is a new service to get current session - https://github.com/oat-sa/generis/blob/develop/common/oatbox/session/SessionService.php